### DISCUSSION TITLE
how to add important in v3

### DISCUSSION THREAD

**Discussion by pincman** [tailwindlabs/tailwindcss#6710] [Help]
In v2, the `!important` is realized by adding the `!` prefix, but the relevant documents can not be found in v3. Can this method still be used?

**Comment by songhn233** [comment]
In v3 important modifier is built-in. v2 should [add `mode: jit`](https://v2.tailwindcss.com/docs/just-in-time-mode#built-in-important-modifier) to enable this feature. You can try in tailwind v3 playground.

![image](https://user-images.githubusercontent.com/47357585/147585993-6ccfcb18-6d76-4b57-8b93-6041b80938d1.png)

**Comment by adamwathan** [comment]
Still works in v3! Good catch on it not being documented, will have to figure out where to explain that.

Where do you think you would expect to see it in the docs?

**Comment by ProPhycient5** [comment]
@pincman Please close the discussion by marking the above reply as `answer`.

**Reply by adamwathan** [reply]
No need to bump year old threads for things like this, I don’t personally care whether people select things as the answer or not, no big deal.

**Reply by ProPhycient5** [reply]
Sure @adamwathan, I just wanted to close various open discussion, which might help other developers.

**Reply by adamwathan** [reply]
Appreciate it! For me though the beauty of discussions is they don’t need to be open or closed like issues, they can just be threads 👍🏻 I tried to convince GitHub to not even support this “answer” feature 😅 so for this community I don’t want to fall into the trap of feeling like that’s something we need to “maintain”.

**Reply by ProPhycient5** [reply]
Sure @adamwathan . That's really nice of you😊. But I felt that some developers can search similar sort of query or problem within discussion and finally they get required answer without wasting any time. Anyways, I loved your vision towards this FOSS.
